libs/escrow-hooks/src/useInvoiceDetails.ts

```diff
@@ -17,7 +17,7 @@ const useInvoiceDetails = ({
     data: invoiceAddress,
     isLoading: invoiceAddressLoading,
     error: invoiceAddressError,
-  } = useContractRead({
+  } = useReadContract({
```

Comment:
@sshmm This file is could use `useInvoiceDetails` from `@smartinvoicexyz/hooks` directly

Lets remove "escrow-gql" and all its uses from DM